client/dead.lua

```diff
@@ -145,8 +150,10 @@ CreateThread(function()
 
             if isDead then
                 if not isInHospitalBed then
-                    if deathTime > 0 then
+                    if deathTime > 0 and IsEmsOnDuty() then
                         DrawTxt(0.93, 1.44, 1.0, 1.0, 0.6, Lang:t('info.respawn_txt', { deathtime = math.ceil(deathTime) }), 255, 255, 255, 255)
+                    elseif not IsEmsOnDuty() then
+                        DrawTxt(0.865, 1.44, 1.0, 1.0, 0.6, Lang:t('info.respawn_revive', { holdtime = hold, cost = Config.BillCost }), 255, 255, 255, 255)
                     else
                         DrawTxt(0.865, 1.44, 1.0, 1.0, 0.6, Lang:t('info.respawn_revive', { holdtime = hold, cost = Config.BillCost }), 255, 255, 255, 255)
```

Comment:
```suggestion
                    else
                        DrawTxt(0.865, 1.44, 1.0, 1.0, 0.6, Lang:t('info.respawn_revive', { holdtime = hold, cost = Config.BillCost }), 255, 255, 255, 255)
```
What's the point of this? This is just duplicate code.

Reply:
Urite